build.gradle

```diff
@@ -2677,9 +2677,6 @@ project(':streams') {
             ':streams:upgrade-system-tests-0101:test',
```

Comment:
please remove  `treams:upgrade-system-tests-0101:test`, `streams:upgrade-system-tests-0102:test` and `streams:upgrade-system-tests-0110:test` as well